examples/integrations/oci/base/identity-se/Dockerfile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2021 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# 1st stage, build the app
+FROM maven:3.6-jdk-11 as build
+
+WORKDIR /helidon
+
+# Create a first layer to cache the "Maven World" in the local repository.
+# Incremental docker builds will always resume after that, unless you update
+# the pom
+ADD ../../pom.xml .
```

Comment:
Not sure how that works, the command documented in the readme is:

```
docker build -t helidon-examples-integrations-oci-listregions-se .
```

So the pom is at the root of the context, meaning that `ADD ../../pom.xml` would fail.

Reply:
fixed